streams/us.m3u

```diff
@@ -109,8 +109,12 @@ https://stream.al-mahdi.tv/hls/1234.m3u8
 http://live.albavision.net:1123/live/albdreams.m3u8
 #EXTINF:-1 tvg-id="Alhurra.us",Alhurra (720p)
 https://mbn-ingest-worldsafe.akamaized.net/hls/live/2038900/MBN_Alhurra_Worldsafe_HLS/master.m3u8
+#EXTINF:-1 tvg-id="AlhurraAlt.us",Alhurra (Alternative) (720p)
+https://ythls.armelin.one/channel/UCyscVWiJELkATSuU-RF2NLg.m3u8
```

Comment:
Not working

Reply:
Sort of. It's the redirector's fault. I didn't notice because it's working for me connecting from Europe. But someone in the middle east told me it doesn't work reliably for them.

I'd imagine this would be an issue with any https://ythls.armelin.one link.

And the official stream is not fine. Watch for more than a few seconds and you will notice.

Reply:
seems fine now